packages/go_router_builder/example/lib/trivial.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
The file name is very generic, what is the specific theme this example is showing?

Reply:
Honestly, trying to have the smallest amount of code that shows how it works. That was my only goal.

Reply:
I could just rename this to `example.dart`

Reply:
will use `simple_example.dart`